pxtlib/docsrender.ts

```diff
@@ -418,7 +418,7 @@ namespace pxt.docs {
 
     export function setupRenderer(renderer: marked.Renderer) {
         renderer.image = function (href: string, title: string, text: string) {
-            const endpointName="makecode-lucas-testing-makecodetempmediaservice-usea";
+            const endpointName="makecodeprodmediaeastus-usea";
```

Comment:
this is the new prod endpoint / not staging endpoint? pretty small but good to give a shout out to things like this in the pr description

Reply:
good point, thanks for feedback!

Reply:
(quick note for next time, pr description typically refers to this portion 
![image](https://user-images.githubusercontent.com/5615930/181314682-6673cf82-e9bb-4d45-becc-ad1aeef15736.png)
where it looked like you changed the title -- not a big deal but short title is usually fine as long as there is further details in description)

Reply:
ah, dang was having trouble finding that, thanks for letting me know!